cycledash/static/js/runs/components/RunsPage.js

```diff
@@ -250,7 +250,8 @@ var RunsTable = React.createClass({
             <th className='caller-name'>Caller Name</th>
             <th className='date'>Submitted</th>
             <th className='num-variants'>Variants</th>
-            <th className='bam-info'>Linked BAMs</th>
+            <th className='linked-bams'>Linked BAMs</th>
+            <th className='run-labels'></th>
```

Comment:
I'd name this and the corresponding React component something like TaskLabels to reflect its new, more limited scope.